docs/conf.py

```diff
@@ -56,8 +56,8 @@
     "sphinx.ext.extlinks",
     "sphinx.ext.intersphinx",
     "sphinx.ext.viewcode",
+    "sphinx.ext.graphviz",
```

Comment:
Let's keep it alphabetical.
```suggestion
    "sphinx.ext.graphviz",
    "sphinx.ext.intersphinx",
    "sphinx.ext.viewcode",
```